gradient/api_sdk/constants.py

```diff
@@ -81,3 +81,13 @@ class RunMode:
         ("MPI", ExperimentType.MPI_MULTI_NODE),
     )
 )
+
+DEPLOYMENT_TYPES_MAP = collections.OrderedDict(
+    (
+        ("TFSERVING", "TFServing"),
+        ("ONNX", "ONNX"),
+        ("CUSTOM", "Custom"),
+        ("FLASK", "Flask"),
+        ("TENSORRT", "TensorRT"),
```

Comment:
Why is it necessary to capitalize these?  Aren't we accepting any case?

Reply:
Yes. It's made to be case insensitive. I've changed the way values are found in this map so the values shown in in terminal don't have to be upper case. Will push to GH in a minute